models/marts/ledger_current_state/accounts_current.sql

```diff
@@ -59,10 +59,32 @@ with
                 > (select max(t.upstream_insert_ts) from {{ this }} as t)
         {% endif %}
     )
+    
+    , account_date as (
+        select
+            account_id
+            , min(batch_run_date) as account_creation_date
+            , min(sequence_number) as min_sequence_number
+        from {{ ref('stg_accounts') }}
+        group by account_id
+    )
+
+    , get_creation_account as (
+        select
+            current_accts.*
+            , account_date.account_creation_date
+            , account_date.min_sequence_number
+        from current_accts
+        left join account_date
```

Comment:
since the `current_accts` table is built from the `stg_accounts`, I don't think there will be a case when an `account_id` exists in `current_accts` and not `account_date`.  Do you think this could be updated to an `inner join`?

Reply:
Yeah, I believe the result would not be different, just used the left join because of good practice since the inner join is used more to filter tables. Should I change it?

Reply:
@vitorweiss yeah, if you can change this to an inner join that would be great. if there ever was a case where a record existed in the `current_accts` table that wasn't in the upstream `stg_accounts`, I would rather drop the record as invalid